crates/typst-library/src/visualize/curve.rs

```diff
@@ -530,3 +530,30 @@ impl Curve {
         Size::new(max_x - min_x, max_y - min_y)
     }
 }
+
+fn point_to_kurbo(point: Point) -> kurbo::Point {
```

Comment:
could you order this in opposite order? so basically
```rust
impl Curve {
  fn contains ..
  fn to_kurbo ..
}

fn point_to_kurbo .. 
```
that would be more in line with the rest of the code base